### DISCUSSION TITLE
error installing Livewire 3 on Windows 11

### DISCUSSION THREAD

**Discussion by wbetalleluzd** [livewire/livewire#5796] [7. Bugs (archived)]
D:\Projects Laravel - Livewire\Projects\tickets.wbdesigner.com  
★ composer require livewire/livewire:^3.0@beta
Info from https://repo.packagist.org: #StandWithUkraine
./composer.json has been updated
Running composer update livewire/livewire
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires livewire/livewire 3.0@beta, found livewire/livewire[dev-throw-error-when-testing-non-livewire-class, ..., dev-bad-hotfix-test, v0.0.1, ..., v0.7.4, v1.0.0, ..., 1.x-dev, v2.0.0, ..., 2.x-dev, v3.0.0-beta.1] but it does not match the constraint.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.

D:\Projects Laravel - Livewire\Projects\tickets.wbdesigner.com  
★

**Comment by joshhanley** [comment]
This issue has been converted to a discussion as we are now using discussions to track bugs and ideas.
See this announcement for more details https://github.com/livewire/livewire/discussions/3147

**Comment by joshhanley** [comment]
@wbetalleluzd thanks for reporting! This is a duplicate of #5775 so we will continue the discussion there. 🙂
